### PR TITLE
Preact to React migration

### PR DESCRIPTION
**Pull request by cedricmessiant**

### CODE REVIEW

**Comment by cedricmessiant** [comment]
[(𝐌) Migration vers React](https://trello.com/c/NVonxC1E/1297-%F0%9D%90%8C-migration-vers-react)

File: src/components/Image/ImageLoader.jsx [hunk 1]
```diff
@@ -126,6 +129,7 @@ class ImageLoader extends React.Component {
   }
 
   render() {
+    return null
```
Comment (Crash--) [inline src/components/Image/ImageLoader.jsx]: to remove

File: src/sharing/ShareModal.jsx [hunk 2]
```diff
@@ -107,7 +110,7 @@ export default class ShareModal extends Component {
 }
 
 ShareModal.propTypes = {
-  document: PropTypes.object.isRequired,
+  document: PropTypes.func.isRequired,
```
Comment (Crash--) [inline src/sharing/ShareModal.jsx]: document, a function? Weird name :/

File: src/drive/mobile/modules/settings/Settings.jsx [hunk 3]
```diff
@@ -51,7 +51,7 @@ class Settings extends Component {
               isDebug={isDebug}
               sendFeedback={this.displayFeedbackForm}
             />
-            <About onTap={this.incrementTapCount} />
+            <About onTap={this.incrementTapCount} t={t} />
```
Comment (Crash--) [inline src/drive/mobile/modules/settings/Settings.jsx]: This is my fault, but I think that doing a translate() within the About component is a better approach?

File: src/sharing/ShareModal.jsx (hunk 2 above)
Comment (cedricmessiant) [reply src/sharing/ShareModal.jsx]: Indeed. I will take a look at this.

**Comment by cedricmessiant** [comment]
Set it to Ready for review to have some news from Travis, but it is not ready yet

**Pull request merged**